app/ocr.py

```diff
@@ -320,26 +324,21 @@ def get_application(self) -> str:
 
     def get_outcome(self) -> str:
         """
-        • Returns the outcome of the case. This will appear after 'ORDER'
-        at the end of the document.
-        """
-        outcomes = {
-            'remanded',
-            'reversal',
-            'dismissed',
-            'sustained',
-            'terminated',
-            'granted',
-            'denied',
-            'returned',
-        }
+        • Returns list of outcome terms from the case in a list. These will appear after 'ORDER' at the end of the document.
+        """
+        outcomes_return = []
+        ordered_outcome = {'ORDER', 'ORDERED'}
+        outcomes_list = ['denied','dismissed','granted','remanded','returned','reversal','sustained','terminated','terninated','vacated']
+        #Interesting edge case in 349320269- typo on 'terminated' present in the pdf: fuzzywuzzy matches terminated to [(terninated, 90)]
         for token in self.doc:
-            if token.text in {"ORDER", "ORDERED"}:
-                start, stop = token.sent.start, token.sent.end + 50
-                outcome = self.doc[start:stop].text.strip().replace("\n", " ")
-                outcome = outcome.split('.')[0]
-                if any(itm.lower() in outcomes for itm in outcome.split()):
-                    return outcome
+            if(str(token) in ordered_outcome):
+                # Can be changed to append on partial match: len(fuzzy_match[0][0]) > 5 and fuzzy_match[0][1] >= 90
+                for n in range(0, len(outcomes_list)):
+                    fuzzy_match = process.extract(outcomes_list[n], self.doc[token.i:token.i+175], limit=1)

```

Comment:
I was trying the fuzzy match since I saw one case had "terninated" instead of "terminated" and wanted to try making a partial match if fuzzywuzzy was 90% sure. However this gave some errors where it would pick up matches like 'dismiss' in trying to match 'dismissed' that were erroneously being added. So I'm leaving it as an exact match only.